Backend/app/alchemist/src/Main.hs

```diff
@@ -67,7 +67,7 @@ getFilePathsForConfiguredApps rootDir = do
 main :: IO ()
 main = do
   putStrLn ("Version " ++ NammaDSL.version)
-  generateAllSpecs <- ("--all" `elem`) <$> getArgs
+  generateAllSpecs <- ("all" `elem`) <$> getArgs
```

Comment:
--all was working for me ..
I was doing 
`, run-generator --all`

Reply:
Your change is  also okay.. working for we 
`, run-generator all`

Reply:
reverted this change, I tested wrong way